src/main/java/org/opentripplanner/routing/api/response/RoutingErrorCode.java

```diff
@@ -0,0 +1,25 @@
+package org.opentripplanner.routing.api.response;
+
+public enum RoutingErrorCode {
+  SYSTEM_ERROR,
```

Comment:
We should throw OTPApplicationException to exit a search, so we do not need the SYSTEM_ERROR here. we do need it on the API enum.